src/audio/dcblock/tune/example_dcblock.m

```diff
@@ -31,13 +31,16 @@ function example_dcblock()
 function dcblock_blob_calculate(prm)
 
 % Set the parameters here
-tplg1_fn = sprintf("../../topology/topology1/m4/dcblock_coef_%s.m4", prm.id); % Control Bytes File
-tplg2_fn = sprintf("../../topology/topology2/include/components/dcblock/%s.conf", prm.id);
+sof_tools = '../../../../tools';
+sof_tplg = fullfile(sof_tools, 'topology');
+sof_ctl = fullfile(sof_tools, 'ctl');
+tplg1_fn = sprintf("%s/topology1/m4/dcblock_coef_%s.m4", sof_tplg, prm.id); % Control Bytes File
+tplg2_fn = sprintf("%s/topology2/include/components/dcblock/%s.conf", sof_tplg, prm.id);
 % Use those files with sof-ctl to update the component's configuration
-blob3_fn = sprintf("../../ctl/ipc3/dcblock/coef_%s.blob", prm.id); % Blob binary file
-alsa3_fn = sprintf("../../ctl/ipc3/dcblock/coef_%s.txt", prm.id); % ALSA CSV format file
-blob4_fn = sprintf("../../ctl/ipc4/dcblock/coef_%s.blob", prm.id); % Blob binary file
-alsa4_fn = sprintf("../../ctl/ipc4/dcblock/coef_%s.txt", prm.id); % ALSA CSV format file
+blob3_fn = sprintf("%s/ipc3/dcblock/coef_%s.blob", sof_ctl, prm.id); % Blob binary file
```

Comment:
`prm.id`? what is `prm` and what is `id`

Reply:
Prm was earlier added to collect the parameters into an easy to pass struct. Id is a string to append to blob file name dcblock/coef_<id>.conf. Current usages are "default" and  like "50hz_48kHz". Currently there are blobs for 20, 30, 40, 50, 80, 100, 200 Hz cut-off @ 48 kHz.